Code/Client/src/components/io/io.js

```diff
@@ -0,0 +1,38 @@
+
+
+export function setUpSocketHandlers(component) {
```

Comment:
io.js is a freaky filename bro, make that shit more informative

Reply:
suggestion? I can't have all that code in the game.js file, it will get sloppy